examples/src/main/java/com/mapbox/navigation/examples/core/NavigationViewFragment.kt

```diff
@@ -0,0 +1,49 @@
+package com.mapbox.navigation.examples.core
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.fragment.app.Fragment
+import com.mapbox.navigation.dropin.MapboxNavigationViewApi
+import com.mapbox.navigation.dropin.NavigationView
+import com.mapbox.navigation.dropin.NavigationViewOptions
+import com.mapbox.navigation.dropin.ViewProvider
+import com.mapbox.navigation.examples.core.databinding.FragmentNavigationViewBinding
+
+class NavigationViewFragment : Fragment() {
+    private lateinit var binding: FragmentNavigationViewBinding
```

Comment:
`binding` should be set to `null` in `onDestroyView` ([docs](https://developer.android.com/topic/libraries/view-binding#fragments)), likely the same for `mapboxNavigationViewApi`. Also we can just use local variables.